<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo14</class>
 <widget class="QDialog" name="dlgDemo14">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>585</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Demo14: Consulta Desconectada de Varios Alumno con Foto FS</string>
  </property>
  <widget class="QLabel" name="lblCodigo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblApellidos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellidos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombres">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombres</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombres">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFechaNacimiento">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Nac</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFechaNacimiento">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFoto">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Foto</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgFoto">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUltimo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrimero">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnterior">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSiguiente">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPosicion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCodigo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblDocumentoIdentidad">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Doc Iden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDocumentoIdentidad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
